apps/dashboard/app/lib/smart_attributes/attributes/auto_job_name.rb

```diff
@@ -36,6 +34,16 @@ def label(*)
       def submit(*)
         { script: { job_name: value } }
       end
+
+      # TODO: need to sanitize the job name for some schedulers
+      def job_name(name)
+        [
+          ENV['OOD_PORTAL'], # the OOD portal id
+          ENV['RAILS_RELATIVE_URL_ROOT'].to_s.sub(%r{^/[^/]+/}, ''), # the OOD app
```

Comment:
Are we just pulling the app or the router and app? I ask because the comment only mentions the app but:
```ruby
irb(main):003:0> "/pun/dev/dashboard".sub(%r{^/[^/]+/}, '')
=> "dev/dashboard"
```
Just wanted to check the intent.

Reply:
It's the router and the app (and the portal and the subapp).

If you look in activejobs right now at OSC you'll see job names like this, so this follows the same implementation that the jobcomposer and batch connect apps currently use.

![image](https://github.com/user-attachments/assets/e139c2cc-2129-4410-abde-dc9d524ba6a7)